test/unit/org/apache/cassandra/tools/cqlsh/CqlshTest.java

```diff
@@ -44,4 +48,29 @@ public void testKeyspaceRequired()
         assertThat(tool.getCleanedStderr(), CoreMatchers.containsStringIgnoringCase("No keyspace has been specified"));
         assertEquals(2, tool.getExitCode());
     }
+
+    @Test
+    public void testUseUnreservedKeywordAsColumnName()
+    {
+        List<String> names = Arrays.asList("access", "datacenters");
+        for (String name : names)
+        {
+            testUseUnreservedKeywordAsColumnName(name);
+            testUseUnreservedKeywordAsColumnName(name.toUpperCase());
+        }
+    }
```

Comment:
nit: I would have put that test into  `CreateTest`. I would not think of looking into CqlshTest for a test on unreserved Keywords.